airbyte-ci/connectors/connectors_qa/tests/unit_tests/test_checks/test_testing.py

```diff
@@ -67,50 +70,50 @@
 THRESHOLD_USAGE_VALUES = ["high", "medium"]
 OTHER_USAGE_VALUES = ["low", "none", "unknown", None, ""]
 
-DYNAMIC_INTEGRATION_TESTS_ENABLED_CASES = [
-  METADATA_CASE_WITH_INTEGRATION_TEST_SUITE_OPTIONS,
-  METADATA_CASE_WITH_INTEGRATION_TEST_SUITE_OPTIONS_NONE_SECRETS,
-  METADATA_CASE_WITH_INTEGRATION_TEST_SUITE_OPTIONS_EMPTY_SECRETS,
-  METADATA_CASE_WITH_INTEGRATION_TEST_SUITE_OPTIONS_NO_SECRETS,
+DYNAMIC_ACCEPTANCE_TESTS_ENABLED_CASES = [
+  METADATA_CASE_WITH_ACCEPTANCE_TEST_SUITE_OPTIONS,
+  METADATA_CASE_WITH_ACCEPTANCE_TEST_SUITE_OPTIONS_NONE_SECRETS,
+  METADATA_CASE_WITH_ACCEPTANCE_TEST_SUITE_OPTIONS_EMPTY_SECRETS,
+  METADATA_CASE_WITH_ACCEPTANCE_TEST_SUITE_OPTIONS_NO_SECRETS,
 ]
 
-DYNAMIC_INTEGRATION_TESTS_DISABLED_CASES = [
+DYNAMIC_ACCEPTANCE_TESTS_DISABLED_CASES = [
     METADATA_CASE_NO_TEST_SUITE_OPTIONS,
     METADATA_CASE_EMPTY_TEST_SUITE_OPTIONS,
     METADATA_CASE_NONE_TEST_SUITE_OPTIONS,
-    METADATA_CASE_MISSING_INTEGRATION_TEST_SUITE_OPTIONS,
+    METADATA_CASE_MISSING_ACCEPTANCE_TEST_SUITE_OPTIONS,
 ]
 
 
-class TestIntegrationTestsEnabledCheck:
+class TestAcceptanceTestsEnabledCheck:
     @pytest.mark.parametrize(
         "cases_to_test, usage_values_to_test, expected_result",
         [
             (
-                DYNAMIC_INTEGRATION_TESTS_DISABLED_CASES + DYNAMIC_INTEGRATION_TESTS_ENABLED_CASES,
+                DYNAMIC_ACCEPTANCE_TESTS_DISABLED_CASES + DYNAMIC_ACCEPTANCE_TESTS_ENABLED_CASES,
                 OTHER_USAGE_VALUES,
-                CheckStatus.PASSED
+                CheckStatus.SKIPPED
             ),
             (
-                DYNAMIC_INTEGRATION_TESTS_ENABLED_CASES,
+                DYNAMIC_ACCEPTANCE_TESTS_ENABLED_CASES,
                 THRESHOLD_USAGE_VALUES,
                 CheckStatus.PASSED
             ),
-            (
-                DYNAMIC_INTEGRATION_TESTS_DISABLED_CASES,
-                THRESHOLD_USAGE_VALUES,
-                CheckStatus.FAILED
-            )
+                (
+                    DYNAMIC_ACCEPTANCE_TESTS_DISABLED_CASES,
+                    THRESHOLD_USAGE_VALUES,
+                    CheckStatus.FAILED
+                )
         ],
     )
     def test_check_always_passes_when_usage_threshold_is_not_met(self, mocker, cases_to_test, usage_values_to_test, expected_result):
         for usage_value in usage_values_to_test:
             for metadata_case in cases_to_test:
                 # Arrange
-                connector = mocker.MagicMock(cloud_usage=usage_value, metadata=metadata_case)
+                connector = mocker.MagicMock(cloud_usage=usage_value, metadata=metadata_case, language=ConnectorLanguage.PYTHON, connector_type="source")
```

Comment:
@alafanechere ❓ why did we specify language here?

Reply:
Because `Check.run` expects it to be set to go through the `applies_to_connector_language` check